### PR TITLE
sql-math-function-abs

### LINKED ISSUE #2216: [Term Entry] SQL Math Functions ABS
### Reason/inspiration (optional)

We would like a new term entry for the Math Function, `ABS`, in SQL.

The entry should go in a file in docs/content/sql/concepts/math-functions/terms/abs/abs.md

The entry should: 

- Explain what the math function `ABS` is in SQL and what it’s used for, starting with an introductory paragraph (2-3 sentences).
- Include SQL database compatibility as a note.
- Links out to entries as needed.

Please refer to the [entry templates (term entry template in particular)](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by nikolaspoczekaj**
### Description

This closes #2216 . Thanks for assigning me.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/sql/concepts/math-functions/terms/abs/abs.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+Title: 'ABS()'
+Description: 'Returns the absolute value of a given number'
```
Comment (yangc95) [inline content/sql/concepts/math-functions/terms/abs/abs.md:3]:
```suggestion
Description: 'Returns the absolute value of a given number.'
```

File: content/sql/concepts/math-functions/terms/abs/abs.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  - 'PostgreSQL'
+  - 'MySQL'
+  - 'Functions'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+The `ABS()` function in SQL returns the absolute value of a given number.
+That means, It effectively converts negative numbers to positive numbers, and leaves positive numbers unchanged.
```
Comment (yangc95) [inline content/sql/concepts/math-functions/terms/abs/abs.md:18]:
```suggestion
The `ABS()` function in SQL returns the absolute value of a given number. That means, it effectively converts negative numbers to positive numbers, and leaves positive numbers unchanged.
```

File: content/sql/concepts/math-functions/terms/abs/abs.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+## Syntax
+
+```sql
+SELECT ABS(number)
+FROM table_name;
+```
+
+## Example
+
+In this example the following data is given:
```
Comment (yangc95) [inline content/sql/concepts/math-functions/terms/abs/abs.md:29]:
```suggestion
In this example, the following data is given in a `transactions` table:
```

**Review by yangc95 — COMMENTED** [review]
Nice one @nikolaspoczekaj! Minor changes here.

Also, can you add a note about which flavors of SQL support this function? I found these: MySQL, PostgreSQL, Oracle and SQL Server but please confirm and add it in as a note.

**Comment by nikolaspoczekaj** [comment]
Thanks @yangc95 . Commited the changes. What do you think ?

File: content/sql/concepts/math-functions/terms/abs/abs.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+The output will be:
+
+| temperature_id | absolute_temp |
+| -------------- | ------------- |
+| 1              | 5.7           |
+| 2              | 18.5          |
+| 3              | 30.2          |
+
+**Note:** `ABS()` is compatible with various SQL database systems such as MySQL, PostgreSQL, Oracle, and SQL Server.
```
Comment (yangc95) [inline content/sql/concepts/math-functions/terms/abs/abs.md:51]: turn this into a block quote

**Review by yangc95 — APPROVED** [review]
Looks good for a second review, just make sure to turn that note into a block quote

**Review by yangc95 — APPROVED** [review]
Looks good for a second review, just make sure to turn that note into a block quote

**Comment by nikolaspoczekaj** [comment]
Done :) @yangc95

File: content/sql/concepts/math-functions/terms/abs/abs.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  - 'SQLite'
+  - 'PostgreSQL'
+  - 'MySQL'
+  - 'Functions'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+The `ABS()` function in SQL returns the absolute value of a given number. That means, it effectively converts negative numbers to positive numbers, and leaves positive numbers unchanged.
```
Comment (caupolicandiaz) [inline content/sql/concepts/math-functions/terms/abs/abs.md:17]:
```suggestion
The **`ABS()`** function in SQL returns the absolute value of a given number. That means, it effectively converts negative numbers to positive numbers, and leaves positive numbers unchanged.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@nikolaspoczekaj Looks good, just left one comment.

**Comment by nikolaspoczekaj** [comment]
@caupolicandiaz thanks. change commited.

**Review by caupolicandiaz — APPROVED** [review]
@nikolaspoczekaj @yangc95 Looks good to me.

**Comment by SSwiniarski** [comment]
@nikolaspoczekaj Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/sql/math-functions/abs

image

**Pull request merged**